crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,6 +14,18 @@
 # limitations under the License.
 #
 
+# Enable proper traceback logging of delayed jobs
+class Delayed
```

Comment:
ah yes, good catch, is a leftover of a hound complaint, it was class `Delayed::Worker` before!